recipes/localhgt/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -eu -o pipefail
+
+mkdir -p $PREFIX/bin
+
+
+#export CPLUS_INCLUDE_PATH="${PREFIX}/include"
+#export LIBRARY_PATH="${PREFIX}/lib"
+#export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+
+#make CC="${GXX}" CFLAGS="${CFLAGS}"
+#make CC="${GXX} ${LDFLAGS}" CFLAGS="${CFLAGS}"
+
+make
```

Comment:
https://github.com/deepomicslab/LocalHGT/blob/f6dda3b789fd717f025f23c1927157073884048d/Makefile#L1 - here `CC` is hardcoded to `g++`.
This line should be `CC ?= g++` so that it could be overwritten with `make CC=$CXX`